### ISSUE TITLE
[BUG] focus indicator not visible in dark mode for grey nav links

### ISSUE THREAD

**Issue by EmmaDawsonDev** [rupali-codes/LinksHub#568]
### Description

Set the site to dark mode. Tab through the site on desktop to the navigation on the left-hand side. Press enter on Frontend to show the categories within then tab through Animations, colors, Design Insp... etc. The focus indicator cannot be seen because it is black on a dark purple background. 

### Any additional information?

This is probably a good first issue so if someone else wants to have a go then feel free. Otherwise, I can fix it.

### What browser are you seeing the problem on?

Chrome

### Checklist

- [X] I have checked the existing issues
- [X] I have read the [Contributing Guidelines](https://github.com/rupali-codes/LinksHub/blob/main/CONTRIBUTING.md)
- [X] I am willing to work on this issue (optional)

**Comment by EmmaDawsonDev** [comment]
https://github.com/rupali-codes/LinksHub/assets/57045550/60917bc8-af93-4b47-8fd0-3d2e75a4fdd0

**Comment by KaKashi1210** [comment]
i fixed it can i get it assigned so i can make pr ?
